app/controllers/people_controller.rb

```diff
@@ -0,0 +1,28 @@
+class PeopleController < ApplicationController
+
+  def index
+    @people = Person.all
+    @task = Task.all
+
+    Task.where(:completed_at => nil).count
+
+    render :index
+
+  end
+
+  def show
+    @task = Task.find(params[:id])
```

Comment:
Do you think that the corresponding ID that comes in from the route would match both the task and the person? (i.e. you use `params[:id]` on line 14 & 17)